reconcile/openshift_resources_base.py

```diff
@@ -566,6 +570,7 @@ def fetch_openshift_resource(
             msg = "could not render template at path {}\n{}".format(path, e)
             raise ResourceTemplateRenderError(msg)
     elif provider == "vault-secret":
+        path = resource["path"]
```

Comment:
is `_locked_log` statement missing after this line?

Reply:
yes it is 🤦

Reply:
[57a3b40](https://github.com/app-sre/qontract-reconcile/pull/2737/commits/57a3b40f079128b2c79d89e22479907596a417e0)

Reply:
shit happens 😬 😂